docs/marketplace/user/features/202108.0/marketplace-cart-notes-feature-overview.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Marketplace Cart Notes feature overview
+description: This document contains concept information for the Marketplace Cart Notes feature.
+template: concept-topic-template
+---
+
+The **Marketplace Cart Notes** feature allows you to include a "Notes" field on the cart page. Buyers can add notes to a particular item or the whole cart, for example, some special instructions about how to prepare and deliver an order.
```

Comment:
as we mention item level, let's please add merchant portal screenshot of an order as well